File: SeriesPrograms/fibonacci.py
def fib(n: int):
    if n == 0:
        return 0
    elif n==1 or n==2:
        return 1
    return fib(n-1) + fib(n-2)

print('Enter the number of terms: ')
n = int(input())
for i in range(n):
    print(fib(i), end=" ")

File: SeriesPrograms/Fibonacci.java
import java.util.*;

public class Fibonacci {
    
    static long fib(int n){
        if(n==0){
            return 0;
        }
        else if(n==1 || n==2){
            return 1;
        }
        return fib(n-1) + fib(n-2);
    }
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter the number of terms: ");
        int n = in.nextInt();
        for(int i=0; i<n; i++){
            System.out.println(fib(i));
        }
    }
}
